arrows/qt/widgets/MetadataView.cxx

```diff
@@ -71,12 +72,8 @@ MetadataViewPrivate::addItem( int id, QString const& keyText )
   {
     auto* const q = this->m_contentWidget;
 
-    auto* const keyLabel = new QLabel {
-      keyText, q
-    };
-    auto* const valueLabel = new qtSqueezedLabel {
-      q
-    };
+    auto* const keyLabel = new QLabel { keyText, q };
```

Comment:
See previous comment. This is even more surprising, although I do get this behavior in C mode.